doozer/doozerlib/assembly_inspector.py

```diff
@@ -315,8 +317,9 @@ def check_group_image_consistency(self, build_inspector: BrewBuildImageInspector
         content_git_url = image_meta.config.content.source.git.url
         if content_git_url:
             # Make sure things are in https form so we can compare
-            content_git_url, _ = self.runtime.get_public_upstream(util.convert_remote_git_to_https(content_git_url))
-            build_git_url = util.convert_remote_git_to_https(build_inspector.get_source_git_url())
+            content_git_url, _ = self.runtime.get_public_upstream(
+                artcommonlib.util.convert_remote_git_to_https(content_git_url))
```

Comment:
we can use `import import artcommonlib.util as artutil` and use shorter syntax here `artutil.convert_remote_git_to_https(content_git_url)`

Reply:
Done in https://github.com/openshift-eng/art-tools/pull/160/commits/472105997483d39d8c8068bd30b200b79b719fdc